test/e2e/framework/deployment_util.go

```diff
@@ -242,3 +244,86 @@ func WaitForDeploymentRevision(c clientset.Interface, d *extensions.Deployment,
 func CheckDeploymentRevisionAndImage(c clientset.Interface, ns, deploymentName, revision, image string) error {
 	return testutils.CheckDeploymentRevisionAndImage(c, ns, deploymentName, revision, image)
 }
+
+func CreateDeployment(client clientset.Interface, replicas int32, podLabels map[string]string, namespace string, pvclaims []*v1.PersistentVolumeClaim, command string) (*extensions.Deployment, error) {
+	deploymentSpec := MakeDeployment(replicas, podLabels, namespace, pvclaims, false, command)
+	deployment, err := client.Extensions().Deployments(namespace).Create(deploymentSpec)
+	if err != nil {
+		return nil, fmt.Errorf("deployment %q Create API error: %v", deploymentSpec.Name, err)
+	}
+	glog.Infof(fmt.Sprintf("Waiting deployment %q to complete", deploymentSpec.Name))
```

Comment:
Is there a reason to be using this vs. framework.Logf?

Reply:
You are right - we should use framework.Logf here for consistency. Will fix.